reference/excel/chartformatstring.md

```diff
@@ -9,7 +9,7 @@ None
 ## Relationships
 | Relationship | Type	|Description| Req. Set|
 |:---------------|:--------|:----------|:----|
-|font|[ChartFont](chartfont.md)|Represents the font properties of chartFormatString object. Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
+|font|[ChartFont](chartfont.md)|Represents the font attributes of chart substring object, such as font name, font size, color . Read-only.|[1.9](../requirement-sets/excel-api-requirement-sets.md)|
```

Comment:
Please add the article "the" before "chart substring. Also add "or" before "color".